web-frontend/src/pages/SelectAccount/SelectAccount.tsx

```diff
@@ -0,0 +1,65 @@
+import LandingPage from '../../layouts/LandingPage/LandingPage';
+import { FiUser, FiPlus } from 'react-icons/fi';
+import { Button } from '@massalabs/react-ui-kit/src/components/Button/Button';
+import { AccountSelectorButton } from '@massalabs/react-ui-kit/src/components/AccountSelector/';
+import { MassaToken } from '@massalabs/react-ui-kit/src/components/Icons/Svg/SvgComponent/MassaToken';
+
+export default function SelectAccount() {
+  const selectProps = {
+    avatar: <FiUser className="text-neutral h-6 w-6" />,
+    accountName: 'account #',
+    icon: <MassaToken size={24} />,
+    amount: '0,000.00',
+  };
+
+  const buttonProps = {
+    onClick: () => {
+      console.log('clicked');
+    },
+    preIcon: <FiPlus className="w-6 h-6" />,
+  };
+
+  const margin24 = 'mb-6';
+  const margin16 = 'mb-4';
+  const defaultFlex = 'flex flex-col justify-center items-center align-center';
+  const buttonFontColor = 'text-neutral';
+  const selectAccountWidth = 'w-[498px]';
```

Comment:
I don't see the interest of margin24, margin16, buttonFontColor, selectAccountWidth

Reply:
WDYT @mazmassa

Reply:
Agree!

I would say if we have only related we can place it directly to the html tag. Only if we have something bigger or with events like: hover, etc...

Reply:
BTW, why we are setting that in 498px!? 🤔  Why not 50% or something related!? w-1/2!?

Reply:
Yes ! I wanted to sync with you to see how we change that ! thanks for the suggestion !